protocol/flatbuffers/rpc.fbs

```diff
@@ -27,7 +27,10 @@ union RpcMessage {
 	OpenSerialRequest,
 	CloseSerialRequest,
 	SetWifiRequest,
-	SerialUpdateResponse
+	SerialUpdateResponse,
+	AutoBoneProcessRequest,
+	AutoBoneProcessStatus,
```

Comment:
Please format your file with spaces, not tabs, at least until I add an autoformatting config to this file to switch the whole thing to tabs as per Eirens request. Feel free to fix the tabs on `SkeletonConfigRequest`, etc too